internal/pod_models.go

```diff
@@ -550,6 +550,9 @@ func configureSidecarContainer(container *corev1.Container, initMode bool, proce
 		if cluster.Spec.Routing.PodIPFamily != nil {
 			sidecarArgs = append(sidecarArgs, "--public-ip-family")
 			sidecarArgs = append(sidecarArgs, fmt.Sprint(*cluster.Spec.Routing.PodIPFamily))
+			if *cluster.Spec.Routing.PodIPFamily == 6 {
```

Comment:
Can we use the `pointer` package here and to deference the setting if set? Otherwise we might get an nil pointer exception.